frontera.md

```diff
@@ -12,25 +12,70 @@ Then execute:
 
 ```
 module load tacc-apptainer
-singularity pull -F docker://seissol/training:latest
-singularity build -f my-training.sif singularity.def
-singularity run my-training.sif
+apptainer pull -F docker://seissol/training:latest
+apptainer build -f my-training.sif singularity.def
+apptainer run my-training.sif
+```
+
+You can also use the automatically generated container after pulling the docker container 
+
+```
+module load tacc-apptainer
+apptainer pull -F docker://seissol/training:latest
+apptainer run training_latest.sif
 ```
 
 You can abort the jupyter lab with Ctrl-C, confirm with `y`.
 Now you should see a directory `seissol-training`.
 This folder should contain four directories for different scenarios.
 
+You can also open the jupyter notebook that runs on Frontera on your local machine with the following steps:
+
+Step 1: change `SHARED_PATH="/your/path/to/container/"` in line 75 of `job.jupyter` to the path where your sigularity container is built.
+Step 2: Run
+```
+sbatch job.jupyter
```

Comment:
sbatch -A <your_project> job.jupyter
should we also need to specify
`-m 30 -N 1 --tasks-per-node 2 -p development`

Reply:
Thank you, Thomas! The rest of the setup for the slurm job is inside the job.jupyter. I will add the -A option in the Readme.